Gems/ROS2/Code/CMakeLists.txt

```diff
@@ -45,7 +45,8 @@ ly_add_target(
 )
 
 target_depends_on_ros2_packages(${gem_name}.Static rclcpp builtin_interfaces std_msgs sensor_msgs nav_msgs urdfdom tf2_ros ackermann_msgs gazebo_msgs control_toolbox)
-
+target_include_directories(${gem_name}.Static PUBLIC "/opt/ros/humble/include/control_toolbox/")
+target_link_libraries(${gem_name}.Static PUBLIC "/opt/ros/humble/lib/libcontrol_toolbox.so")
```

Comment:
question: Should these be updated to a relative path as opposed to an absolute path? Or is this standard and okay on Linux with ros? (tagging @lumberyard-employee-dm in case they have a suggestion on this approach too)

Reply:
I think this is temporary in the Draft PR

Reply:
I see understood, thanks for confirming 👍